src/main/kotlin/mate/academy/LetterRemover.kt

```diff
@@ -1,5 +1,10 @@
 package mate.academy
 
-class LetterRemover {
-    // implement ...
+object LetterRemover {
```

Comment:
Please check task description, there should be companion object.

In case if there are some difficulties with checkstyle (Detect error) please try to run via IDE (see 
![Screenshot 2024-04-05 141028](https://github.com/mate-academy/kt-companion-object/assets/83113910/46c76062-dfb1-46fc-bee9-ddd189d56f99)
![Screenshot 2024-04-04 123155](https://github.com/mate-academy/kt-companion-object/assets/83113910/a50d2cda-9d48-4242-9ce7-54ae2568bbde)
attached screenshots)

If required result is obtained and test are passed you may ignore detect error and send to review